custom/icds_reports/ucr/data_sources/dashboard/delivery_forms.json

```diff
@@ -107,6 +107,15 @@
           "property_path": ["form", "meta", "timeEnd"]
         }
       },
+      {
+        "column_id": "completed_on",
```

Comment:
curious how you chose this column name? is subjective but i'd have favored a more explicit `timeend_with_time` or something to indicate it's the same field as the one above but with a different data type.

Reply:
oww, I can do that as well. But I chose completed_on with thinking of that this is the time when the form was completed on phone. But, hearing your option. I think that sounds more relevant to me. Will change it

Reply:
cool, thanks. yeah i think the problem with `completed_on` is that you could just replace it with `timeend` in your sentence above and that would also be true. So it's confusing what the difference is between the two properties.